tools/generator/src/clean.js

```diff
@@ -0,0 +1,20 @@
+const rimraf = require("rimraf");
+const { FEATURES } = require('./feature-names');
+const { updateBuildFile } = require('./build-file');
+const { updateNgModule, removeRoutesFromNgModule, removeRoutesFromAppComponentHtml } = require('./ng-module');
+
+module.exports = function () {
+ FEATURES.map(feature => `src/app/${feature.path}`).forEach(featPath => {
+
+  console.log(`DELETE ${featPath}`);
+  rimraf.sync(featPath);
+
+ });
+
+ updateBuildFile('src/app/BUILD.bazel', { mappedFeatureList: [] });
```

Comment:
don't add this extra complexity of touching non-generated files

Reply:
Let me know if you disagree with https://github.com/angular/angular-bazel-example/pull/358#discussion_r273439639 I will then remove all this complexity 🙂

Reply:
Yes, I do disagree, we have no tests for this code and no time to maintain it so I want all the mechanisms to stay as simple as possible. Flexibility is probably not desirable here.